content/en/docs/installation/uninstall.md

```diff
@@ -25,3 +25,45 @@ Delete the `NOTATION_LIBEXEC` directory to remove the additional binaries, inclu
 **IMPORTANT:** `notation` automatically creates the `NOTATION_CONFIG` directory if it does not exist. If you remove the `NOTATION_CONFIG` directory, `notation` will recreate the directory the next time you run a command.
 
 Delete the `NOTATION_CONFIG` directory to remove the configuration files.
+
+## Remove the test key and self-signed certificate
+
+You can generate a sample test key and self-signed certificate using `notation cert generate-test`. For example:
+
+```console
+notation cert generate-test --default "wabbit-networks.io"
+```
+
+At this time, that key and self-signed certificate can't be removed using `notation key delete` and `notation cert delete`. To remove this key and self-signed certificate, remove the key from the signing list and manually delete the key and certificate files.
+
+Use `notation key ls` to show the location of the key and certificate files. For example:
+
+```console
+notation key ls
+NAME                   KEY PATH                                           CERTIFICATE PATH                                 ID   PLUGIN NAME
+* wabbit-networks.io   NOTATION_CONFIG/localkeys/wabbit-networks.io.key   NOTATION_CONFIG/localkeys/wabbit-networks.io.crt
+```
+
+Use `notation key delete` to remove the key from the signing list. For example:
+
+```console
+notation key delete wabbit-networks.io
+```
+
+Remove the key file using the path from `notation key ls`. For example:
+
+```console
```

Comment:
could we give a hint here for customers who do not know the the value of "NOTATION_CONFIG" variable, they could just do "notation certificate list" to learn the path for their "NOTATION_CONFIG" variable.

Reply:
There is a callout on line 13: https://github.com/notaryproject/notaryproject.dev/pull/164/files#diff-ebf28445e02ae412163e2fde43e76bee51e880abab8449ac8b4b863fec8d4354L13